app/views/ImporterAddressConfirmationView.scala.html

```diff
@@ -0,0 +1,87 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import controllers.routes._
+@import models.Mode
+
+@this(
+        main_template: MainTemplate,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+        govukSelect: GovukSelect,
+        link: components.link
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@pageHint = {
+    <h2 class="govuk-heading-m" style="margin-bottom: 0.3em">
+        @messages("importerAddress.hint")
+        <p/>
+    </h2>
+}
+
+@h4_1 = {
+    <strongr >
+        NE98 1AE
```

Comment:
@somaraja this is the postcode that gets pulled through from the previous screen user answers. Can we just leave this blank for now as we haven't done the plumbing yet?

Reply:
@lmugnai As advised changes incorporated.